charts/tidb-cluster/values.yaml

```diff
@@ -73,6 +73,12 @@ pd:
     [replication]
     location-labels = ["region", "zone", "rack", "host"]
 
+  # pd Service
+  # we can only specify clusterIP and loadBalancerIP now
+  service:
+    clusterIP: "None"
```

Comment:
Looks like that this variable is not rendered in the template, actually, I recommend to leave the helm chart as-is because we are migrating from helm to CRD #1121